pathway-analysis/qusage_meta_analysis.Rmd

```diff
@@ -253,7 +273,10 @@ northcott_mat <- northcott_mat[, northcott_metadata_df$refinebio_accession_code]
 #### Expression data
 
 ```{r}
-robinson_expression_file <- file.path("data", "GSE37418", "GSE37418.tsv")
+robinson_expression_file <- file.path(data_dir,
```

Comment:
Same comment here. If you decide to do a `northcott_dir` should also do `robinson_dir`.